tests/scenarios/github_test_13/cmd1

```diff
@@ -0,0 +1,7 @@
+source: ubuntu-1-deployment
```

Comment:
We had a similar test case that failed in GH action since for some reason ping didn't result in SOCK_RAW operation! We had to comment that testcase. CC @daemon1024

Reply:
Yes @nyrahul. I and @daemon1024  were trying to find other tools that could help us test these scenarios.
`traceroute` is another one that uses SOCK_RAW. However in certain environments like Cent OS, even `traceroute` doesn't use SOCK_RAW.
So prolly we'll need to comment this one out too.

Reply:
But I can see that the CI has passed for this PR. So I wonder why the tests have not failed ?? CC @daemon1024

Reply:
They were made to be skipped in the following commit, so the CI is back to being passing. Ref https://github.com/kubearmor/KubeArmor/pull/685/commits/4715e6430b915e8c9c3f1f637301845561250ae3